modules/Core/classes/Group_Sync/MinecraftGroupSyncInjector.php

```diff
@@ -26,14 +26,17 @@ public function getColumnType(): string {
     }
 
     public function shouldEnable(): bool {
-        return count($this->getSelectionOptions()) > 0;
+        return Util::getSetting('group_sync_mc_server') != 0 && count($this->getSelectionOptions()) > 0;
```

Comment:
This `shouldEnable` method is called during a group sync broadcast, meaning the Minecraft injector will be ignored fully if they have not select a server to get group sync groups from - I think this is the best way forward